TiDB Cloud Docs: Add documentation for new Recovery Group feature (#17425)
pingcap/docs: tidb-cloud/tidb-cloud-billing-recovery-group.md

---
title: Recovery Group Billing
summary: Learn about billing for recovery groups in TiDB Cloud.
---

# Recovery Group Billing

> TiDB Cloud bills for recovery groups based on the deployed size of your TiKV nodes in the primary cluster of the recovery group. When you [create a recovery group](/tidb-cloud/recovery-group-get-started.md) for a cluster, you can select the primary cluster for the recovery group. The larger the TiKV configuration, the higher the cost for recovery group protection.

Reviewer: pls add data processing part

Author: Added in https://github.com/pingcap/docs/pull/17425/commits/4eb89d97980e7babd25e5577794c22ef6665f406

Reviewer: I think we should not tell the customer how we make the pricing of data processing  - “This data processing cost includes the cross-region and cross-AZ traffic charges”, we just tell them there is a data processing cost charge by GB is enough.

Author: Okay, updated in https://github.com/pingcap/docs/pull/17425/commits/6bd256f1a842e600e42f1266960a32ca3307a898

> TiDB Cloud bills for recovery groups based on the deployed size of the TiKV cluster that is the primary cluster in the recovery group. When you [create a recovery group](/tidb-cloud/recovery-group-get-started.md) for a cluster, you can select the primary cluster that the recovery group will be associated with. The larger the TiKV configuration, the higher the cost for recovery group protection.

Reviewer 2: ```suggestion
TiDB Cloud bills for recovery groups based on the deployed size of your TiKV nodes in the primary cluster of the recovery group. When you [create a recovery group](/tidb-cloud/recovery-group-get-started.md) for a cluster, you can select the primary cluster for the recovery group. The larger the TiKV configuration, the higher the cost for recovery group protection.
```

> ## Price
> 
> To learn about the supported regions and the price of TiDB Cloud for each recovery group, see [Recovery Group Cost](https://www.pingcap.com/tidb-cloud-pricing-details/#recovery-group-cost).

Reviewer 2: ```suggestion
## Pricing

To learn about the supported regions and the pricing for TiDB Cloud recovery groups, see [Recovery Group Cost](https://www.pingcap.com/tidb-cloud-pricing-details/#recovery-group-cost).
```

---
title: Recovery Group Billing
summary: Learn about billing for recovery groups in TiDB Cloud.
---

# Recovery Group Billing

TiDB Cloud bills for recovery groups based on the deployed size of your TiKV nodes in the primary cluster of the recovery group. When you [create a recovery group](/tidb-cloud/recovery-group-get-started.md) for a cluster, you can select the primary cluster for the recovery group. The larger the TiKV configuration, the higher the cost for recovery group protection.

> TiDB Cloud also bills for data processing per GB basis. The data processing price varies depending on whether the data is replicated to a secondary cluster in another region, or within the same region.

Reviewer 2: ```suggestion
TiDB Cloud also bills for data processing per GiB basis. The data processing price varies depending on whether the data is replicated to a secondary cluster in another region, or within the same region.
```
